package.json

```diff
@@ -157,7 +157,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn format-staged && yarn lint-staged"
+      "pre-commit": "yarn format-staged && yarn lint-staged && yarn run link-check"
```

Comment:
if we don't use localhost for the base url it will fail, right?

Reply:
also means that it fails if local server is not running ... probably too much complication to run it on pre-commit

Reply:
The CLI script works completely fine on pre-commit with no local server, with the exception that new root-relative links that don't exist will be reported as nonexistent. 
To mitigate root-relative issues, I left the pre-commit as just an informative check that doesn't block commits. This way, editors can see ahead of time if a link fails, but can also continue if it's just root-relative links that will be fulfilled on the preview with the real GH Actions check.

The ability to optionally run this against the dev server would be good though, so I'll keep that in mind.